messaginginapp/src/test/java/io/customer/messaginginapp/ModuleMessagingInAppTest.kt

```diff
@@ -148,4 +108,100 @@ internal class ModuleMessagingInAppTest : JUnitTest() {
         // verify that the module's dismissMessage method was called
         assertCalledOnce { inAppMessagesProviderMock.dismissMessage() }
     }
+
+    @Test
+    fun `when screen viewed event occurs, gist provider sets current route`() {
```

Comment:
I know this is a common naming convention used in Android, but it seems inconsistent with our codebase. I would appreciate if we could follow the same naming convention at least across the file.

Reply:
yeah that was added by mistake